### ISSUE TITLE
Need a Landing Page

### ISSUE THREAD

**Issue by Sahil1gupta** [Sahil1gupta/CheatNotes#7]
**Hello Folks, We a Need landing page for cheatnotes web application**

**Comment by yashsarnaik22** [comment]
if you just need a homepage(index.html), I can do this.
assign this to me.

**Comment by avezqureshi14** [comment]
@Sahil1gupta  If you need Single Page Application (SPA) using REACT JS which will be much more better than landing page along with deployment then you can assign this to me

**Comment by Sahil1gupta** [comment]
@avezqureshi14 our application is fully based on naive HTML,CSS, BOOTSTRAP and JavaScript it would be great if you could design the landing page using these frontend languages.

**Comment by Rohitpal0123** [comment]
@Sahil1gupta bro remove that practice.py page.

**Comment by Sahil1gupta** [comment]
> if you just need a homepage(index.html), I can do this. assign this to me.

yes you can but try to make landing page more elegant. if you can add animation into it then you're good to go.
